### PR TITLE
Script rules replaced with remove-attr scriptlets.

### PR DESCRIPTION
**Pull request by stanislav-atr**
https://github.com/AdguardTeam/AdguardFilters/issues/87667

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1389,7 +1388,12 @@ nekopoi.web.id#%#AG_onLoad(function() { document.body.oncontextmenu = document.b
...
-poisonous-raspberry-fields.blogspot.com#%#AG_onLoad(function(){var a=document.getElementsByTagName("body")[0];a&&(a.removeAttribute("onmousedown"),a.removeAttribute("onselectstart"),a.removeAttribute("ondragstart"));a=document.querySelectorAll(".post-body[ondragstart][onmousedown][onselectstart]");for(var b=0;b<a.length;b++)a[b].removeAttribute("onmousedown"),a[b].removeAttribute("onselectstart"),a[b].removeAttribute("ondragstart")});
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onselectstart", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "ondragstart", "body")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown", ".post-body[ondragstart][onmousedown][onselectstart]")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onselectstart", ".post-body[ondragstart][onmousedown][onselectstart]")
+poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "ondragstart", ".post-body[ondragstart][onmousedown][onselectstart]")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1396]: `poisonous-raspberry-fields.blogspot.com#%#//scriptlet("remove-attr", "onmousedown|onselectstart|ondragstart", "body, .post-body[ondragstart][onmousedown][onselectstart]")`. For the similar rules do the same thing.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -1401,7 +1405,9 @@ ouo.io#%#AG_onLoad(function() { setTimeout(function() { var el = document.queryS
 tokubatch.com#%#AG_abortOnPropertyRead('nocontext');
 tokubatch.com#%#AG_abortOnPropertyRead('disable_copy');
 chiaseapk.com#%#AG_onLoad(function() { document.oncontextmenu = document.onselectstart = function() { return true; }; });
-siyasetcafe.com#%#AG_onLoad(function() { setTimeout(function() { var b = document.body; if(b) { b.removeAttribute("onselectstart"); b.removeAttribute("oncontextmenu"); b.removeAttribute("ondragstart"); } }, 300); });
+dragonball-tube.com,onepiece-tube.com,fairytail-tube.org,baho.com.tr,futboltv24.net,wholehk.com,siyasetcafe.com#%#//scriptlet("remove-attr", "onselectstart", "body")
+dragonball-tube.com,onepiece-tube.com,fairytail-tube.org,baho.com.tr,futboltv24.net,wholehk.com,siyasetcafe.com#%#//scriptlet("remove-attr", "oncontextmenu", "body")
+dragonball-tube.com,onepiece-tube.com,fairytail-tube.org,baho.com.tr,futboltv24.net,wholehk.com,siyasetcafe.com#%#//scriptlet("remove-attr", "ondragstart", "body")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1410]: `dragonball-tube.com,onepiece-tube.com,fairytail-tube.org,baho.com.tr,futboltv24.net,wholehk.com,siyasetcafe.com#%#//scriptlet("remove-attr", "onselectstart|oncontextmenu|ondragstart", "body")`

File: AnnoyancesFilter/sections/tweaks.txt [hunk 3]
```diff
@@ -1415,18 +1421,22 @@ tunovelaligera.com#%#Object.defineProperties(window,{sccopytext:{value:function(
...
 мвд.рф#%#AG_onLoad(function() { setTimeout(function() { document.ondragstart=document.onselectstart=document.oncontextmenu=document.ondragend=document.ondrop=document.oncopy=document.onpaste = null; }, 300); });
 topeuropix.net,livetvcafe.me#%#Object.defineProperty(window, 'clickIE', { get: function() { return; } });
-runningnews.gr#%#Object.defineProperty(window, 'disableselect', { get: function() { return; } }); AG_onLoad(function(){AG_each('[oncontextmenu]',function(el){el.removeAttribute('oncontextmenu');});AG_each('[ondragstart]',function(el){el.removeAttribute('ondragstart');});AG_each('[onselectstart]',function(el){el.removeAttribute('onselectstart');});AG_each('img[onmousedown]',function(el){el.removeAttribute('onmousedown');});});
+runningnews.gr#%#//scriptlet("remove-attr", "oncontextmenu", "[oncontextmenu]")
+runningnews.gr#%#//scriptlet("remove-attr", "ondragstart", "[ondragstart]")
+runningnews.gr#%#//scriptlet("remove-attr", "onselectstart", "[onselectstart]")
+runningnews.gr#%#//scriptlet("remove-attr", "onmousedown", "img[onmousedown]")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1427]: Make one rule.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 4]
```diff
@@ -1415,18 +1421,22 @@ tunovelaligera.com#%#Object.defineProperties(window,{sccopytext:{value:function(
...
+lanjutkeun.blogspot.com#%#//scriptlet("remove-attr", "oncontextmenu", "body")
+lanjutkeun.blogspot.com#%#//scriptlet("remove-attr", "onkeydown", "body")
 tvlive.cc,worldfree4ufilm.com,pelispedia.tv#%#AG_onLoad(function() { document.oncontextmenu = function(){return true}; });
 kseries.co#%#Object.defineProperties(window, { clickIE: { get: function() { return; } }, disableselect: { get: function() { return; } } });
-dragonball-tube.com,onepiece-tube.com,fairytail-tube.org#%#AG_onLoad(function() { setTimeout(function() { document.getElementsByTagName("Body")[0].removeAttribute("oncontextmenu"); document.getElementsByTagName("Body")[0].removeAttribute("onselectstart"); document.getElementsByTagName("Body")[0].removeAttribute("ondragstart"); }, 300); });
-hindistopin.com,hindinewstop.com#%#AG_onLoad(function() { setTimeout(function() { document.getElementsByTagName("Body")[0].removeAttribute("oncontextmenu"); document.getElementsByTagName("Body")[0].removeAttribute("onkeydown"); }, 300); });
+hindistopin.com,hindinewstop.com#%#//scriptlet("remove-attr", "oncontextmenu", "body")
+hindistopin.com,hindinewstop.com#%#//scriptlet("remove-attr", "onkeydown", "body")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1439]: Make one rule.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 5]
```diff
@@ -1441,18 +1451,19 @@ gaypornmasters.com#%#AG_defineProperty('oncontextmenu', {value:true}, document);
...
 worldfree4ufilm.com,spy-soft.net,z-news.link#%#Object.defineProperty(window, 'disableSelection', { get: function() { return; } });
 itbox.ua,vokrug.tv#%#AG_onLoad(function() { document.oncopy = function() {}; });
 kinoshek.net,kinogo-720p.ru,allhit.org,korp.tv#%#Object.defineProperty(window, 'preventSelection', { get: function() { return function(){}; } });
 guvenlilink.blogspot.com#%#AG_onLoad(function() { document.oncontextmenu=function(){return true}; });
 latinohentai.com,7days2die.info,f4freesoftware.com#%#Object.defineProperties(document,{oncontextmenu:{set:function(){}},ondragstart:{set:function(){}}});
-7lafa.com#%#AG_onLoad(function() {setTimeout(function(){document.getElementsByTagName('body')[0].removeAttribute("oncopy"); document.getElementsByTagName('body')[0].removeAttribute("onselectstart");}, 300);});
+7lafa.com#%#//scriptlet("remove-attr", "oncopy", "body")
+7lafa.com#%#//scriptlet("remove-attr", "onselectstart", "body")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1463]: Make one rule.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 6]
```diff
@@ -1466,7 +1477,8 @@ picrelated.com#%#AG_onLoad(function() { window.CHV.obj.config.image.right_click
 pornhdx.tv#%#AG_onLoad(function() { setTimeout(function() {$(document).unbind('contextmenu');}, 300); });
 skidrowreloaded.com#%#var _={get:function(){return function(){}}};Object.defineProperties(window,{clickIE4:_,disableSelection:_});
 thegreatdaily.com#%#Object.defineProperty(window, 'clearSelection', { get: function() { return []; } });
-webspor2.com,biyografi.info,seiya-saiga.com#%#AG_onLoad(function() { setTimeout(function() { document.getElementsByTagName("Body")[0].removeAttribute("oncontextmenu"); document.getElementsByTagName("Body")[0].removeAttribute("onselectstart"); }, 300); });
+webspor2.com,biyografi.info,seiya-saiga.com#%#//scriptlet("remove-attr", "oncontextmenu", "body")
+webspor2.com,biyografi.info,seiya-saiga.com#%#//scriptlet("remove-attr", "onselectstart", "body")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1481]: Make one rule.

File: EnglishFilter/sections/general_extensions.txt [hunk 7]
```diff
@@ -1640,7 +1639,8 @@ wetpussy.sexy#%#AG_onLoad(function(){document.querySelectorAll('a[href^="/cc/out
 api.youtube-mp3.org.in#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/50106
 p4link.com,oceantech.xyz#%#//scriptlet("abort-on-property-read", "tampilkanUrl")
-p4link.com,oceantech.xyz#%#AG_onLoad(function(){if(document.querySelector("#countdown")){var b=new MutationObserver(function(){var a=document.querySelector('a[class="btn btn-success btn-lg get-link"][target="_blank"]');a&&(a.classList.remove("get-link"),a.removeAttribute("target"),b.disconnect())});b.observe(document,{childList:!0,subtree:!0})}});
+p4link.com,oceantech.xyz#%#//scriptlet("remove-attr", "target", "a[class='btn btn-success btn-lg get-link'][target='_blank']")
+p4link.com,oceantech.xyz#%#//scriptlet("remove-class", "get-link", "a[class='btn btn-success btn-lg get-link'][target='_blank']")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:1643]: Make one rule.

File: EnglishFilter/sections/general_extensions.txt [hunk 8]
```diff
@@ -2794,7 +2794,8 @@ animewow.org,animenova.org#%#AG_onLoad(function() { window.showAd = function() {
 ! https://github.com/AdguardTeam/AdguardFilters/issues/12794
 imagefap.com#%#Object.defineProperty(window, 'Buu', { get: function() { return; } });
 ! https://github.com/AdguardTeam/AdguardFilters/issues/12705
-telecharger-youtube-mp3.com#%#AG_onLoad(function(){AG_each('a[href][onclick^="setTimeout(function(){ window.open("][onclick*="clksite.com"]',function(el){el.removeAttribute('onclick'); el.removeAttribute('href');});});
+telecharger-youtube-mp3.com#%#//scriptlet("remove-attr", "onclick", "a[href][onclick^='setTimeout(function(){ window.open('][onclick*='clksite.com']")
+telecharger-youtube-mp3.com#%#//scriptlet("remove-attr", "href", "a[href][onclick^='setTimeout(function(){ window.open('][onclick*='clksite.com']")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:2798]: Make one rule.

File: EnglishFilter/sections/general_extensions.txt [hunk 9]
```diff
@@ -3081,9 +3083,10 @@ saavn.com#%#!function(a,b){b=new MutationObserver(function(){a.classList.contain
 ! https://github.com/AdguardTeam/AdguardFilters/issues/5388
 csgoconfigs.com#%#document.cookie = "AD_SHOWN=true";
 ! https://forum.adguard.com/index.php?threads/littlebyte-net.22689/
-littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');a.removeAttribute('target');})})
+littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');})})
+littlebyte.net#%#//scriptlet("remove-attr", "target   ", "a[onclick^='getActionContent']")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:3087]: `littlebyte.net#%#//scriptlet('remove-attr', 'target', 'a[onclick^="getActionContent"]')`. Is this rule necessary?

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 7 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/general_extensions.txt:1643]: Different scriptlets

File: AnnoyancesFilter/sections/tweaks.txt (hunk 3 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/tweaks.txt:1427]: Different selectors.

File: EnglishFilter/sections/general_extensions.txt (hunk 9 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/general_extensions.txt:3087]: Issue is not found on our forum and `littlebyte.net` main page gives no acces to other pages, so i am not able to test this one.

File: EnglishFilter/sections/general_extensions.txt [hunk 10]
```diff
@@ -3084,7 +3083,7 @@ saavn.com#%#!function(a,b){b=new MutationObserver(function(){a.classList.contain
 csgoconfigs.com#%#document.cookie = "AD_SHOWN=true";
 ! https://forum.adguard.com/index.php?threads/littlebyte-net.22689/
 littlebyte.net#%#AG_onLoad(function(){AG_each('a[onclick^="getActionContent"]',function(a){a.setAttribute('href','#');})})
-littlebyte.net#%#//scriptlet("remove-attr", "target   ", "a[onclick^='getActionContent']")
+littlebyte.net#%#//scriptlet("remove-attr", "target", "a[onclick^='getActionContent']")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:3086]: Use external single quotes https://github.com/AdguardTeam/AdguardFilters/pull/87851#discussion_r666235076

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**